datadog/resource_datadog_timeboard.go

```diff
@@ -579,7 +590,7 @@ func resourceDatadogTimeboardCreate(d *schema.ResourceData, meta interface{}) er
 		return fmt.Errorf("Failed to create timeboard using Datadog API: %s", err.Error())
 	}
 	d.SetId(strconv.Itoa(timeboard.GetId()))
-	return nil
+	return resourceDatadogTimeboardRead(d, meta)
```

Comment:
Why this change?

Reply:
This is a best practice, as mentioned in [1]. It's necessary in this PR as line [2] actually creates an attribute on the TF schema that doesn't exist and thus persisting it in the state would make TF fail. This is precisely one of the reasons why calling the read function at the end of create/update is recommended, as it overwrites changes like this one.

[1] https://www.terraform.io/docs/extend/best-practices/detecting-drift.html#update-state-after-modification
[2] https://github.com/terraform-providers/terraform-provider-datadog/pull/215/files#diff-22ba5b0fbe5d1b3cdfe783aeb61bf51aR755

Reply:
Ahh I see, thanks for the helpful explanation/links 🙇